src/serial.c

```diff
@@ -858,6 +858,7 @@ static void evaluateCommand(void)
         mcfg.board_align_yaw = read16(); // board_align_yaw
         mcfg.currentscale = read16();
         mcfg.currentoffset = read16();
+        mcfg.motor_pwm_rate = read16();

```

Comment:
sure but if we're going to change size let's add about 10 or so bytes of
dummy shit for later expansion.

not on irc now
On Jan 18, 2015 1:44 AM, "luggi" <notifications@github.com> wrote:

> In src/serial.c
> <https://github.com/multiwii/baseflight/pull/238#discussion-diff-23126446>
> :
>
> > @@ -858,6 +858,7 @@ static void evaluateCommand(void)
> >          mcfg.board_align_yaw = read16(); // board_align_yaw
> >          mcfg.currentscale = read16();
> >          mcfg.currentoffset = read16();
> > +        mcfg.motor_pwm_rate = read16();
>
> i think it should be added to the configurator -> see irc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/multiwii/baseflight/pull/238/files#r23126446>.
>